esp-hal/src/dma/mod.rs

```diff
@@ -48,6 +48,137 @@
 
 use core::{fmt::Debug, marker::PhantomData, ptr::addr_of_mut, sync::atomic::compiler_fence};
 
+trait Word {}
```

Comment:
We should seal this

Reply:
ah it's not public but sealing won't hurt